scripts/ci/in_container/check_environment.sh

```diff
@@ -131,34 +119,21 @@ function resetdb_if_requested() {
     return $?
 }
 
-if [[ -n ${BACKEND:=} ]]; then
-    echo "==============================================================================================="
-    echo "             Checking backend: ${BACKEND}"
-    echo "==============================================================================================="
 
-    set +e
+echo "==============================================================================================="
+echo "             Checking integrations and backends"
+echo "==============================================================================================="
+if [[ -n ${BACKEND:=} ]]; then
```

Comment:
set +e / -e should be around that - check_db_connection exits with '1" in case of error.

Reply:
I don't think it is like you say. This function communicates through a global variable - EXIT_CODE, not "real" exit codes. We can clean up some code in this file to make it clearer. Thanks for comment.

Reply:
The problem is that you can get some extra output in this check. Bash function might either pass output by printing result in the output or returning exit code. I. This case I chose exit code in other to avoid ambiguity of printed stdout by the check command. It is of course possible to avoid it but I figured exit code is better, simpler and avoids unexpected behaviour. 

But feel free to change it if you take care about the ambiguity .